pkg/operator/ceph/cluster/watcher.go

```diff
@@ -262,11 +262,22 @@ func listRBDPV(listPVs *corev1.PersistentVolumeList, cluster *cephv1.CephCluster
 	var listRbdPV []corev1.PersistentVolume
 
 	for _, pv := range listPVs.Items {
-		// Ignore PVs that support multinode access (RWX, ROX), since they can be mounted on multiple nodes.
-		if pvSupportsMultiNodeAccess(pv.Spec.AccessModes) {
+		// Skip if pv is not provisioned by CSI
+		if pv.Spec.CSI == nil {
+			logger.Infof("pv %q is not provisioned by CSI", pv.Name)
 			continue
 		}
+
 		if pv.Spec.CSI.Driver == fmt.Sprintf("%s.rbd.csi.ceph.com", cluster.Namespace) {
+			// Ignore PVs that support multinode access (RWX, ROX), since they can be mounted on multiple nodes.
+			if pvSupportsMultiNodeAccess(pv.Spec.AccessModes) {
+				continue
+			}
+			if pv.Spec.CSI.VolumeAttributes["staticVolume"] == "true" || pv.Spec.CSI.VolumeAttributes["pool"] == "" || pv.Spec.CSI.VolumeAttributes["imageName"] == "" {
+				logger.Infof("skipping, static pv %q", pv.Name)
```

Comment:
i would suggest keeping this as debug log as well, this log will be repeated depending on the number of static PVC.